com.ibm.wala.cast.js/src/main/java/com/ibm/wala/cast/js/callgraph/fieldbased/WorklistBasedOptimisticCallgraphBuilder.java

```diff
@@ -53,14 +53,18 @@ public class WorklistBasedOptimisticCallgraphBuilder extends FieldBasedCallGraph
 
   private FlowGraphBuilder builder;
 
+  private Integer bound = -1;
```

Comment:
If you are going to use `-1` as the sentinel value indicating there is no bound, then this field can just be of type `int`.  I suggested type `Integer` in case you wanted to use `null` to indicate no bound.  But -1 is fine.  Please add Javadoc on this field indicating what it is used for and what it means for it to have a value of `-1`.

Also please make the field `final`; we don't want it updated after construction.